app/_components/commodities/Form.tsx

```diff
@@ -191,19 +195,27 @@ const Form: React.FC<FormProps> = ({ onSubmitHandler, isLoading }) => {
           ) : (
             <FormLabel marginY="auto">Selling</FormLabel>
           )}
-          <Stack paddingX={8} spacing={8} direction="row" mt={8}>
-            <Select
-              variant="customSelect"
-              id="buying"
-              marginY={'auto'}
-              onChange={(e) => {
-                setIsBuying(e.target.value === 'buying');
-              }}
-            >
-              <option value="buying">Buy</option>
-              <option value="selling">Sell</option>
-            </Select>
-
+          <Text>I am looking to:</Text>
```

Comment:
We could move this into a `FormLabel` and replace the current 'buying'/'selling' one since it seems fairly redundant

Reply:
Perhaps just move this 'I am looking to:' into the form label and drop the current 'Buying' as it seems unnecessary now.

Reply:
Oops, this was an artifact of the merge conflict. fixed.